example/assets/css/app.scss

```diff
@@ -1 +1,34 @@
 @import '~harmonium/scss/app';
+
+
+/**
+* The input_group_field helper produces slightly different markup
+* from the React component FieldGroup.Field. These styles can be
+* used to make the appropriate adjustments
+*/
+.rev-InputGroup-fieldWrapper{
+  width: 100%;
+
+  input, select, textarea {
+    @include input-style;
+    border-radius: 0;
+    &:first-child {
+      border-bottom-left-radius: $input-radius;
+      border-top-left-radius: $input-radius;
+    }
+    &:last-child {
+      border-bottom-right-radius: $input-radius;
+      border-top-right-radius: $input-radius;
+    }
+  }
+
+  input {
+    @extend .rev-Input;
```

Comment:
These aren't strictly necessary and may even be a bad idea. Without them, you'd have to make sure whatever you put in there has the appropriate `rev-` class already or things look weird.